.github/workflows/on-push-master-publish-chart.yml

```diff
@@ -0,0 +1,86 @@
+name: Publish helm chart
+
+on:
```

Comment:
Any reason to have this duplicated?

Reply:
The idea is that we never publish a chart release if linting has failed. It's technically possible to commit a broken chart, ignore the linting failure, and merge to main. The duplication prevents accidentally publishing broken charts.

Reply:
I would like to prevent this duplication otherwise we need to maintain multiple files with the same content.